moyopy/python/moyopy/interface.py

```diff
@@ -59,7 +60,7 @@ def from_py_obj(struct: Structure | ase.Atoms) -> moyopy.Cell:
         Returns:
             moyopy.Cell: The converted Moyo cell
         """
-        if isinstance(struct, ase.Atoms):
+        if isinstance(struct, (ase.Atoms, MSONAtoms)):
             return MoyoAdapter.from_atoms(struct)
```

Comment:
Do we need to update type annotations for `MoyoAdapter.from_atoms` and `MoyoAdapter.from_py_obj` as well?

Reply:
good call